doc/imv.1

```diff
@@ -1,131 +1,125 @@
-.TH IMV 1 "" "" "multimedia"
-.SH NAME
-imv \- image viewer
-.SH SYNOPSIS
-.nf
-\fBimv\fP [-rfaudlh] [-n <NUM|PATH>] [-b COLOR] [-e FONT:SIZE] [-t SECONDS] [images...]
-.fi
-.sp
-.SH DESCRIPTION
-.sp
-\fBimv\fP is a X11/Wayland image viewer aimed at users of tiling window managers.
+.Dd $Mdocdate: December 30 2015$
```

Comment:
The '$' sign is my mistake - I'm too much used to CVS expansion.

"BSD General Commands Manual" in the header is expanded from default name of section 1 of your operating system. "BSD" in the footer is expanded from your operating system name. Apparently your groff is misconfigured. What distro do you use?